src/components/button/bl-button.css

```diff
@@ -76,15 +81,24 @@
 .button:focus-visible::after {
   border: 2px solid var(--main-color);
   border-radius: var(--bl-border-radius-l);
-  content: "";
+  content: '';
   position: absolute;
   inset: -4px;
 }
 
+.loading-icon {
+  animation: spin 1s linear infinite;
+  font-size: var(--icon-size);
+}
+
 :host ::slotted(bl-icon) {
   font-size: var(--icon-size);
 }
 
+:host([loading]) ::slotted(bl-icon) {
+  display: none;
+}
```

Comment:
Apparently this doesn't work with `icon` attribute, only works by setting icon slot.

<img width="182" alt="image" src="https://user-images.githubusercontent.com/127687/222536606-67f7a1f3-b87b-4009-b0ea-9c59bb8eaec4.png">

Maybe we can cover icon slot with an element to target it regardless of where it set from.